tasks/managed/upload-sbom-to-atlas/upload-sbom-to-atlas.yaml

```diff
@@ -229,6 +229,7 @@ spec:
             -H "transfer-encoding: chunked" \
             -H "content-type: application/json" \
             --data "@$supported_version_of_sbom" \
-            "$bombasticApiUrl/api/v1/sbom?id=$sbom_id"
+            "$bombasticApiUrl/api/v1/sbom?id=$sbom_id" \
+            || (echo "SBOM upload to Atlas has failed!")
```

Comment:
Why the `()`? That opens a subshell, is that necessary? It seems to work just fine without it.

Also, would be nice to have a test for this - to mock curl to return a non-zero code and confirm the task still passes.

Reply:
Removed the `()` and updated the `mocks.sh` to return the error code for curls to Atlas.

Reply:
Thanks. But it would be preferable if this was a separate test - the way to do it is create a new test pipeline and in it use a special string for the curl call - in this case the easiest would probably be to use a file like `"$sbomsDir/spdx_minimal_fail_2_2.json"` and look for `spdx_minimal_fail` in the mock or something like that.

Reply:
No problem. Added new test.